apps/server/src/modules/system/system.module.ts

```diff
@@ -1,12 +1,13 @@
-import { Module } from '@nestjs/common';
 import { IdentityManagementModule } from '@infra/identity-management/identity-management.module';
-import { SystemRepo } from '@shared/repo';
-import { SystemService } from '@modules/system/service/system.service';
+import { Module } from '@nestjs/common';
+import { LegacySystemRepo } from '@shared/repo';
+import { SystemRepo } from './repo';
+import { LegacySystemService, SystemService } from './service';
 import { SystemOidcService } from './service/system-oidc.service';
 
 @Module({
 	imports: [IdentityManagementModule],
-	providers: [SystemRepo, SystemService, SystemOidcService],
-	exports: [SystemService, SystemOidcService],
+	providers: [LegacySystemRepo, LegacySystemService, SystemOidcService, SystemService, SystemRepo],
+	exports: [LegacySystemService, SystemOidcService, SystemService],
```

Comment:
I like that you created the repo directly in the module and renamed the entities but i dont understand why you didnt moved the old one instead of copying it. You have already touched all places where you used the legacy repo because of the renaming. As i can see, both repos are do the same logic and we only have two services to use the different repos, right? ...Because of that i would prefer just to have one repo and one service. They are using the same entity and you are using the same factory everywhere for both repos.

Reply:
Yet both are completely different. The one repo uses the BaseRepo and the old domain object that we introdced and the other one uses the new DomainObject. The difference in services is because one uses the System domain object and the other one is based on the SystemDto.

Reply:
Alright, then could you create a refactoring ticket for that and link it in the deprecations?